pandas/tests/groupby/test_apply.py

```diff
@@ -866,3 +866,25 @@ def fct(group):
         [[1.0, 2.0], [3.0], [np.nan]], index=pd.Index(["a", "b", "none"], name="A")
     )
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    "values",
+    [
+        {
+            "a": [1, 1, 1, 2, 2, 2, 3, 3, 3],
+            "b": [1, pd.NA, 2, 1, pd.NA, 2, 1, pd.NA, 2],
+        },
+        {"a": [1, 1, 2, 2, 3, 3], "b": [1, 2, 1, 2, 1, 2]},
+    ],
+)
+def test_apply_mean_with_nullable_integer(values):
```

Comment:
Is mean the only aggregation this affects?

Reply:
It also affects (at least) `var` and `std` (I think whenever the aggregation can return a float). I'll add some more tests.